src/main/java/org/jetbrains/appenders/JsonLayout.java

```diff
@@ -520,20 +528,19 @@ public void activateOptions() {
         if (includedFields != null) {
             String[] included = SEP_PATTERN.split(includedFields);
             for (String val : included) {
-                renderedFields.add(Field.fromValue(val));
+                fieldLabels.add(val);
             }
         }
-        if (excludedFields != null) {
-            String[] excluded = SEP_PATTERN.split(excludedFields);
-            for (String val : excluded) {
-                renderedFields.remove(Field.fromValue(val));
-            }
-        }
-
         if(renamedFieldLabels != null) {
             for(String fieldLabel : renamedFieldLabels) {
                 String[] field = PAIR_SEP_PATTERN.split(fieldLabel);
-                updateFieldLabel(field[0], field[1]);
+                fieldLabels.update(field[0], field[1]);
```

Comment:
do you like to check it has exactly 2 entries? It will be not good it logger crashes with an exception